views/rosters.ejs

```diff
@@ -29,6 +30,31 @@
                 </div>
                 <%});%>
             </div>
+            <hr>
+            <h1>Our Support Teams</h1>
```

Comment:
Support Teams should be on a separate page so there isn't so much content to scroll through. Could be in a dropdown menu from Rosters header listing Gaming Roster and Support Roster.